src/test/tests.ts

```diff
@@ -234,6 +241,31 @@ describe('SubscriptionManager', function() {
     });
   });
 
+  it('can use an async filter function', function(done) {
```

Comment:
The publish below is for `Filter2` which [as defined above](https://github.com/apollographql/graphql-subscriptions/blob/d4dfbedca6fec24ddd6794d212adab5cf5ac9977/src/test/tests.ts#L125) uses `setTimeout` to delay returning the result.